redbpf/src/lib.rs

```diff
@@ -820,6 +832,43 @@ impl<K: Clone, V: Clone> Iterator for MapIter<'_, '_, K, V> {
     }
 }
 
+impl StackTrace<'_> {
+    pub fn new(map: &Map) -> StackTrace<'_> {
+        StackTrace {
+            base: map
+        }
+    }
+
+    pub fn get(&mut self, id: libc::c_int) -> Option<BpfStackFrames> {
+        unsafe {
+            let mut value = MaybeUninit::uninit();
+
+            let ret = bpf_sys::bpf_lookup_elem(
+                self.base.fd,
+                &id as *const libc::c_int as *mut libc::c_int as _,
+                value.as_mut_ptr() as *mut _
+            );
+
+            if ret == 0 {
+                Some(value.assume_init())
+            } else {
+                None
+            }
+        }
+    }
+
+    pub fn remove(&mut self, id: libc::c_int) -> bool {
```

Comment:
`HashMap` has a `delete()` that is consistent with the bpf terminology, so I'd prefer using that convention instead of `remove`.